src/main/java/it/pagopa/pagopa/apiconfig/config/ResponseValidator.java

```diff
@@ -0,0 +1,49 @@
+package it.pagopa.pagopa.apiconfig.config;
+
+import it.pagopa.pagopa.apiconfig.exception.AppException;
+import lombok.extern.slf4j.Slf4j;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.AfterReturning;
+import org.aspectj.lang.annotation.Aspect;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validator;
+import java.util.Set;
+
+@Aspect
+@Component
+@Slf4j
+public class ResponseValidator {
+
+    @Autowired
+    private Validator validator;
+
+
+    /**
+     * This method validates the response annotated with the {@link javax.validation.constraints}
+     *
+     * @param joinPoint not used
+     * @param result the response to validate
+     */
+    @AfterReturning(pointcut = "execution(* it.pagopa.pagopa.apiconfig.controller.*.*(..))", returning = "result")
+    public void validateResponse(JoinPoint joinPoint, Object result) {
```

Comment:
why did you add this ?
To verify `@NotNull` field ?

Reply:
Yes, to verify `@NotNull` in the responses. In this way, we can detect immediately if our response doesn't follow the swagger definition (ex: description null).

Reply:
Yes, to verify Annotation in the response class